package.json

```diff
@@ -1,40 +1,39 @@
 {
-  "name": "revconf2017-app",
-  "version": "1.0.0",
-  "homepage": "https://github.com/revolutionva/website2017-app#readme",
-  "bugs": {
-    "url": "https://github.com/revolutionva/website2017-app/issues"
-  },
-  "license": "ISC",
-  "main": "index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://iambriansreed@github.com/revolutionva/website2017-app.git"
-  },
-  "scripts": {
-    "start": "node app.js",
-    "postinstall": "node postinstall.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "dependencies": {
-    "adm-zip": "latest",
-    "basic-auth": "^1.1.0",
-    "body-parser": "^1.16.1",
-    "convict": "^2.0.0",
-    "ejs": "^2.5.6",
-    "express": "^4.14.1",
-    "fs-extra": "^2.0.0",
-    "lodash": "^4.17.4",
-    "marked": "^0.3.6",
-    "request": "^2.79.0"
-  },
-  "devDependencies": {
-    "csvtojson": "^1.1.4",
-    "eslint": "^3.16.1",
-    "jimp": "^0.2.27",
-    "commander": "latest"
```

Comment:
Good catch! I updated the Node engine version too because would've probably broken Heroku at version 6.